src/main/webapp/app/exam/manage/exam-management.service.ts

```diff
@@ -22,16 +21,15 @@ import { EntityTitleService, EntityType } from 'app/shared/layouts/navbar/entity
 import { ExamExerciseStartPreparationStatus } from 'app/exam/manage/student-exams/student-exams.component';
 import { Exercise } from 'app/entities/exercise.model';
 
-type EntityResponseType = HttpResponse<Exam>;
-type EntityArrayResponseType = HttpResponse<Exam[]>;
+export type EntityResponseType = HttpResponse<Exam>;
+export type EntityArrayResponseType = HttpResponse<Exam[]>;
```

Comment:
Why is this needed? EntityResponseType is more meant as a shorter way to write the types in this file, but should not be used by the outside (since there many response types are possible).

Reply:
It is commonly used like this throughout the TS codebase, so I thought I apply this pattern. Don't have to, of course, could just use the aliased type `HttpResponse<Exam>` to do the type annotation.

Reply:
I'm talking about the added `export`. In this file it makes sense to have a shortcut type, but it should not be used from the outside.

Reply:
Yea I know, I also mean that, as the `EntityResponseType` is exported from a lot of other services, just search for `export type EntityResponseType` in the codebase, gives me 29 matches. But I'll change that to just using `HttpResponse<Exam>` then :)

Reply:
Gone